makers/engineer_i.md

```diff
@@ -0,0 +1,76 @@
+Engineer I
+==========
+
+As an entry-level engineer, you're at the start of your career, often fresh out of college or a programming bootcamp. You have a solid foundation of technical knowledge and a strong desire to learn.
+
+## Qualifications
+
+- Solid understanding of core CS concepts.
+- Familiarity with programming fundamentals: data structures, classes, debuggers, etc.
+- Connection to Mariana Tek's mission and values.
+
+## Responsibilities & Expectations
+
+<!-- TODO: Add summary -->
```

Comment:
👋 What blockers do we foresee to outlining these responsibilities?

Reply:
Mainly how slowly I write.